pkg/controllers/corecluster/corecluster_controller.go

```diff
@@ -215,6 +219,52 @@ func (r *CoreClusterReconciler) ensureCoreClusterControlPlaneInitializedConditio
 	return nil
 }
 
+// setControllerConditionsToNormal sets the CoreClusterController conditions to the normal state.
+func (r *CoreClusterController) setControllerConditionsToNormal(ctx context.Context, log logr.Logger) error {
+	co, err := r.GetOrCreateClusterOperator(ctx)
+	if err != nil {
+		return fmt.Errorf("failed to get cluster operator: %w", err)
+	}
+
+	conds := []configv1.ClusterOperatorStatusCondition{
+		operatorstatus.NewClusterOperatorStatusCondition(operatorstatus.CoreClusterControllerAvailableCondition, configv1.ConditionTrue, operatorstatus.ReasonAsExpected,
+			"Core Cluster Controller works as expected"),
```

Comment:
What if it's not available? AFAICT there's no way for this controller to say, oops, I have an error, I'm not available.

Is there a reason, after initial setup, where going available false is valid?